api/s2n.h

```diff
@@ -1882,6 +1882,18 @@ S2N_API extern uint64_t s2n_connection_get_delay(struct s2n_connection *conn);
  */
 S2N_API extern int s2n_connection_set_cipher_preferences(struct s2n_connection *conn, const char *version);
 
+/**
+ * Signals the connection to do a key_update at the next possible opportunity.
+ * 
+ * @param conn The connection object to trigger the key update on.
+ * @param peer_update_requested Indicates if a key update should also be requested 
+ * of the peer. If false, then only the sending keys of `conn` will be updated. If
+ * true, then the sending keys of conn will be updated AND the peer will be requested
+ * to update their sending key. Note that s2n-tls currently only supports 
+ * `peer_update_requested` being set to `false`.
```

Comment:
I would call this out a bit more by saying that setting it to true will result in an error